images/oc-build-deploy-dind/openshift-templates/backup-schedule.yml

```diff
@@ -32,17 +32,29 @@ parameters:
     description: Registry where Images are pushed to
     required: true
   - name: BACKUP_SCHEDULE
-    description: Schedule of the Backup in Cron format
+    description: Schedule of the Backup in Cron format or auto schedule format
     required: true
   - name: CHECK_SCHEDULE
-    description: Schedule of the Backup Check in Cron format
+    description: Schedule of the Backup Check in Cron format or auto schedule format
     required: true
   - name: PRUNE_SCHEDULE
-    description: Schedule of the Backup Prune in Cron format
+    description: Schedule of the Backup Prune in Cron format or auto schedule format
     required: true
   - name: BAAS_BUCKET_NAME
     description: Which bucket the backups for this project should go to
     required: true
+  - name: MONTHLY_BACKUP_RETENTION
+    description: How many monthly backups should be retained after pruning
+    required: true
+  - name: WEEKLY_BACKUP_RETENTION
+    description: How many weekly backups should be retained after pruning
+    required: true
+  - name: DAILY_BACKUP_RETENTION
+    description: How many daily backups should be retained after pruning
+    required: true
+  - name: LAGOON_ENVIRONMENT_TYPE
+    description: What type of environment this is (production/development)
+    required: true
```

Comment:
I don't see us inject this as parameters anywhere?

Reply:
Another good catch. Will fix